spec/operators/inequality/color.hrx

```diff
@@ -0,0 +1,106 @@
+<===> boolean/true/input.scss
```

Comment:
Rather than have separate inequality tests that wholly duplicate the equality tests, it's probably better to write a helper mixin that verifies that two values are equal according to both `==` and `!=`. I think it's reasonable to delay this until https://github.com/sass/sass-spec/issues/1418 as well, in which case maybe just leave out inequality specs for now.